### DISCUSSION TITLE
Summary panel (in data explorer) reopens after changing tabs

### DISCUSSION THREAD

**Discussion by Eiriksen** [posit-dev/positron#5562] [Q&A]
After having closed the [summary panel](https://positron.posit.co/data-explorer.html) in data explorer, if I then switch to view another tab and then switch back to the data explorer tab, the summary panel is open again. 

Seems like the data explorer is not remembering that the summary panel has been closed after switching tabs. 

Anyone else experiencing this?

**Comment by anbrav0** [comment]
I have also been experiencing this, i've been following #2823 to see if there is any changes

**Comment by juliasilge** [comment]
Thanks for this report! 🙌 I can reproduce this bug and I opened https://github.com/posit-dev/positron/issues/5572 to track and prioritize a fix.

**Comment by kv9898** [comment]
This is probably also linked to my issue #4979.
